pkg/exchange/max/exchange.go

```diff
@@ -325,11 +341,21 @@ func (e *Exchange) queryClosedOrdersByLastOrderID(
 	return types.SortOrdersAscending(orders), nil
 }
 
-func (e *Exchange) queryClosedOrdersByTime(ctx context.Context, symbol string, since, until time.Time) (orders []types.Order, err error) {
+func (e *Exchange) queryClosedOrdersByTime(ctx context.Context, symbol string, since, until time.Time, orderByType maxapi.OrderByType, limit uint) (orders []types.Order, err error) {
 	if err := e.closedOrderQueryLimiter.Wait(ctx); err != nil {
 		return orders, err
 	}
 
+	// there is since limit for closed orders API
+	sinceLimit := time.Date(2018, time.January, 1, 0, 0, 0, 0, time.Local)
```

Comment:
This time can be moved to package level to be shared with other functions.

Reply:
will use getLaunchDate